piksi_tools/interpolate_event_positions.py

```diff
@@ -0,0 +1,146 @@
+"""
+interpolate_event_trigger.py allows user to analyze log files
+with external event triggers.
+"""
+
+from sbp.client.loggers.json_logger import JSONLogIterator
+from sbp.ext_events import SBP_MSG_EXT_EVENT
+import csv
+
+def lin_interp(o, n, otow, ntow, ttow):
+  """
+  Linearly interpolates distance.
```

Comment:
You Should probably say that the interpolation is precise to the millisecond in this comment